src/debugAdapter/goDebug.ts

```diff
@@ -268,9 +269,11 @@ class Delve {
 					connectClient(port, host);
 				}
 			});
+			let that = this;
```

Comment:
I am not sure why, but "this" in line 275 does not point to Delve object, it points to the ChildProcess which is why I had to resort to `that = this`